docs/plugins.md

```diff
@@ -33,7 +35,7 @@ When plugins cannot be found automatically, you can load them with:
 
 Prettier expects each of `pluginSearchDirs` to contain `node_modules` subdirectory, where `@prettier/plugin-*`, `@*/prettier-plugin-*` and `prettier-plugin-*` will be searched. For instance, this can be your project directory or the location of global npm modules.
 
-Providing at least one path to `--plugin-search-dir`/`pluginSearchDirs` turns off plugin autoloading in the default directory (i.e. `node_modules` above `prettier` binary).
+To turn off plugin autoloading, use `{ pluginSearchDirs: false }`.
```

Comment:
```suggestion
To turn off plugin autoloading, use `{ pluginSearchDirs: false }` in JavaScript code or `--no-plugin-search` when using Prettier CLI.
```

Not sure about exact wording, but maybe it’s a bit better than having `pluginSearchDirs: false` separately from `--no-plugin-search`? WDYT?